lib/FusionInventory/Agent/Config.pm

```diff
@@ -350,6 +350,45 @@ sub logger {
     };
 }
 
+sub getTargets {
+    my ($self, %params) = @_;
+
+    my @targets = ();
+
+    # create target list
+    if ($self->{local}) {
+        FusionInventory::Agent::Target::Local->require()
+            or die "Can't load local target library\n";
```

Comment:
I don't see much point in handling gracefully loading failure for internal components. Or we should do it everywhere else.
BTW, if you dynamically load those classes here, you can probably kill static loading from FusionInventory::Agent